### PR TITLE
Fix rifts in limbo not leading anywhere

### PR DESCRIPTION
**Pull request by noi2coco**

### CODE REVIEW

**Comment by austanss** [comment]
Aren't most rifts in limbo intentionally designed to not lead anywhere? Some rifts do... most rifts don't. Isn't that by design?

File: src/main/java/org/dimdev/dimdoors/block/DimensionalPortalBlock.java [hunk 1]
```diff
@@ -114,7 +114,7 @@ public void onBlockAdded(BlockState state, World world, BlockPos pos, BlockState
 	}
 
 	private static void portalTick(World world, BlockPos pos, BlockState state, EntranceRiftBlockEntity e) {
-		if (world.isClient || e.getDestination() != null) {
+		if (world.isClient || e.getDestination().getLocation() != null) {
```
Comment (CreepyCre) [inline src/main/java/org/dimdev/dimdoors/block/DimensionalPortalBlock.java:117]: I believe the destination may be null. Should probably test for both the destination being null and its location being null.

**Review by CreepyCre — DISMISSED** [review]

File: src/main/java/org/dimdev/dimdoors/block/DimensionalPortalBlock.java (hunk 1 above)
Comment (noi2coco) [reply src/main/java/org/dimdev/dimdoors/block/DimensionalPortalBlock.java:117]: The destination is initialized to NoneTarget and I didn't find a place where it is set to null.
https://github.com/DimensionalDevelopment/DimDoors/blob/5bda2b8017cbc558cbeb20989f414149a001b384/src/main/java/org/dimdev/dimdoors/block/entity/RiftData.java#L9-L10

Comment (Waterpicker) [reply src/main/java/org/dimdev/dimdoors/block/DimensionalPortalBlock.java:117]: > 
> 
> The destination is initialized to NoneTarget and I didn't find a place where it is set to null.
> 
> https://github.com/DimensionalDevelopment/DimDoors/blob/5bda2b8017cbc558cbeb20989f414149a001b384/src/main/java/org/dimdev/dimdoors/block/entity/RiftData.java#L9-L10

Heheheh. I was very meticulous when doing the NoneTarget conversion. ^^

**Comment by tzarbomba64** [comment]
i believe thats the point. its limbo. a place between voids and worlds. meaning the rifts are connecting to themselves. where only bathing in reality itself can free you. it seems like limbo basically was the aether at some point in the lore [im very overthinking of lore] but got corrupted potentially when tey attempted at finding the overworld but broke the door. creating a rift which just kept expanding..

**Comment by Waterpicker** [comment]
Closing because mute point for now since I removed limbo gateways in the gateway rework.

**Pull request closed without merging**